### PR TITLE
CON-2790 CON-2805 Updated application response to contain application type directly

### PR DESCRIPTION
**Pull request by ChrisJWoodcock**

### CODE REVIEW

File: src/SFA.DAS.AssessorService.Api.Types/Models/Apply/ApplicationResponse.cs [hunk 1]
```diff
@@ -21,44 +22,10 @@ public class ApplicationResponse
...
-        public bool IsSequenceActive(int sequenceNo)
-        {
-            // a sequence can be considered active even if it does not exist in the ApplyData, since it has not yet been submitted and is in progress.
-            return ApplyData?.Sequences?.Any(x => x.SequenceNo == sequenceNo && x.IsActive) ?? true;
-        }
-
         public bool IsWithdrawalApplication => IsStandardWithdrawalApplication || IsOrganisationWithdrawalApplication;
 
-        public bool IsStandardWithdrawalApplication => GetApplicationType() == ApplicationTypes.StandardWithdrawal;
+        public bool IsStandardWithdrawalApplication => ApplicationType == ApplicationTypes.StandardWithdrawal;
```
Comment (kola-samuel) [inline src/SFA.DAS.AssessorService.Api.Types/Models/Apply/ApplicationResponse.cs:27]: We should also have  IsStandardApplication and  IsSOrganizationApplication fields

**Review by kola-samuel — CHANGES_REQUESTED** [review]
More Application type Fields required

File: src/SFA.DAS.AssessorService.Api.Types/Models/Apply/ApplicationResponse.cs (hunk 1 above)
Comment (ChrisJWoodcock) [reply src/SFA.DAS.AssessorService.Api.Types/Models/Apply/ApplicationResponse.cs:27]: There are properties for it - I hadn't added them here yet as I didn't have a use - there isn't an organisation application as it cant exists, there is a combined application [(organisation + financial + standard), (organisation + standard), (financial + standard)] or a standard application

**Comment by ChrisJWoodcock** [comment]
> More Application type Fields required

Covered that in my comment above

**Review by cofaulco — APPROVED** [review]

**Review by kola-samuel — APPROVED** [review]

**Pull request merged**